crypto-ffi/bindings/android/build.gradle.kts

```diff
@@ -76,6 +91,48 @@ android {
     }
 }
 
+val processedResourcesDir = layout.buildDirectory.dir("processedResources").get().asFile
+
+fun registerCopyJvmBinaryTask(target: String, jniTarget: String, include: String = "*.so"): TaskProvider<Copy> =
+    tasks.register<Copy>("copy-$target") {
+        group = "uniffi"
+        from(projectDir.resolve("../../../target/$target/release"))
+        include(include)
+        into(processedResourcesDir.resolve(jniTarget))
+    }
+
+val copyBinariesTasks = listOf(
+    registerCopyJvmBinaryTask("aarch64-linux-android", "arm64-v8a"),
+    registerCopyJvmBinaryTask("armv7-linux-androideabi", "armeabi-v7a"),
+    registerCopyJvmBinaryTask("x86_64-linux-android", "x86_64")
+)
+
+project.afterEvaluate {
+    tasks.getByName("mergeReleaseJniLibFolders") { dependsOn(copyBinariesTasks) }
+    tasks.getByName("mergeDebugJniLibFolders") { dependsOn(copyBinariesTasks) }
+}
+
+tasks.withType<ProcessResources> {
+    dependsOn(copyBinariesTasks)
+}
+
+tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile> {
+    dependsOn(copyBindings)
+}
+
+tasks.withType<Jar> {
+    dependsOn(copyBindings)
+}
+
+tasks.withType<Test> {
```

Comment:
What's not working here?

Reply:
The origin of that comment seems to be here https://github.com/wireapp/core-crypto/commit/268db77c42edd2dcbc8fad33dec9d530936c2403, which created the whole file with the comment intact.